### PR TITLE
add test for snli dataset

### PR DESCRIPTION
**Pull request by faneshion**

### CODE REVIEW

File: matchzoo/datasets/snli/load_data.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 _url = "https://nlp.stanford.edu/projects/snli/snli_1.0.zip"
 
 
-def load_data(stage='train', task='ranking', target_label='entailment'):
+def load_data(stage='train', task='classification', target_label='entailment'):
```
Comment (uduse) [inline matchzoo/datasets/snli/load_data.py]: The default task is usually `ranking` across the entire project.

File: tests/unit_test/test_datasets.py [hunk 2]
```diff
@@ -4,13 +4,17 @@
 def test_load_data():
     train_data = mz.datasets.wiki_qa.load_data('train', task='ranking')
     assert len(train_data) == 20360
+    train_data, _ = mz.datasets.wiki_qa.load_data('train', task='classification')
+    assert len(train_data) == 20360
```
Comment (bwanglzu) [inline tests/unit_test/test_datasets.py]: this line didn't test anything new

File: matchzoo/datasets/snli/load_data.py (hunk 1 above)
Comment (faneshion) [reply matchzoo/datasets/snli/load_data.py]: This is dataset is originally a classification problem. We will use it to demonstrate how to use matchzoo to accomplish classification tasks.

File: tests/unit_test/test_datasets.py (hunk 2 above)
Comment (faneshion) [reply tests/unit_test/test_datasets.py]: This line is to test the dataset generation for `classification`, the line above is for `ranking`.

**Pull request merged**